### PR TITLE
feat: Update Property

### PR DESCRIPTION
**Pull request by duolok**
Host can now see a list of his properties and update a property of his choice. Pictures are not being updated.

image
image

### CODE REVIEW

File: app/src/main/res/navigation/base_navigation.xml [hunk 1]
```diff
@@ -54,11 +65,16 @@
...
+        android:id="@+id/nav_my_properties"
+        android:name="com.example.lunark.fragments.updateProperty.MyPropertiesFragment"
+        android:label="My properties"
+        tools:layout="@layout/fragment_favorite_properties" />
     <fragment
         android:id="@+id/nav_reports"
         android:name="com.example.lunark.fragments.ReportsFragment"
         android:label="Reports"
-        tools:layout="@layout/fragment_reports" />
+        tools:layout="@layout/fragment_my_properties" />
```
Comment (MomirMilutinovic) [inline app/src/main/res/navigation/base_navigation.xml]: Why was this changed?
```suggestion
        tools:layout="@layout/fragment_reports" />
```

File: app/src/main/res/navigation/base_navigation.xml [hunk 2]
```diff
@@ -54,11 +65,16 @@
         android:name="com.example.lunark.fragments.FavoritePropertiesFragment"
         android:label="Favorite properties"
         tools:layout="@layout/fragment_favorite_properties" />
+    <fragment
+        android:id="@+id/nav_my_properties"
+        android:name="com.example.lunark.fragments.updateProperty.MyPropertiesFragment"
+        android:label="My properties"
+        tools:layout="@layout/fragment_favorite_properties" />
```
Comment (MomirMilutinovic) [inline app/src/main/res/navigation/base_navigation.xml]: Did you mean:
```suggestion
        tools:layout="@layout/fragment_my_properties" />
```

**Review by MomirMilutinovic — APPROVED** [review]
Looks good for now.
It seems that the type of property and availability entries are not populated when the edit form is opened.

File: app/src/main/res/navigation/base_navigation.xml (hunk 1 above)
Comment (duolok) [reply app/src/main/res/navigation/base_navigation.xml]: Bob Ross would say it was a happy little accident.

**Comment by duolok** [comment]
> Looks good for now. It seems that the type of property and availability entries are not populated when the edit form is opened.

I couldn't get it to work.
